environment_dev.yml

```diff
@@ -57,5 +57,6 @@ dependencies:
       - trl>=0.5.0
       - sentence-transformers
       - rich!=13.1.0
+      - textdescriptives>=2.7.0
```

Comment:
Can you also include the dependency under `pyproject.toml` in the "integrations" extra?

Reply:
Yes, added under `integrations`.